Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioOccluder.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Audio.Influencers
+{
+    /// <summary>
+    /// Class that implements <see cref="IAudioInfluencer"/> to provide an audio occlusion effect, similar
+    /// to listening to sound from outside of an enclosed space.
+    /// </summary>
+    /// <remarks>
+    /// Ensure that all sound emitting objects have an attached <see cref="AudioInfluencerController"/>. 
+    /// Failing to do so will result in the desired effect not being applied to the sound.
+    /// </remarks>
+    [DisallowMultipleComponent]
+    public class AudioOccluder : MonoBehaviour, IAudioInfluencer
+    {
+        /// <summary>
+        /// Frequency above which sound will not be heard after applying occlusion.
+        /// Setting this value to 22000.0 effectively disables the effect.
+        /// </summary>
+        /// <remarks>
+        /// Chaining occluders will result in the lowest of the cutoff frequencies being applied to the sound.
+        /// The CutoffFrequency range is 0.0 - 22000.0 (0 - 22kHz), inclusive.
+        /// The default value is 5000.0 (5kHz).
+        /// </remarks>
+        [Tooltip("Frequency above which sound will not be heard after applying occlusion.")]
+        [Range(10.0f, 22000.0f)]
+        [SerializeField]
+        private float cutoffFrequency = 5000.0f;
+        public float CutoffFrequency
+        {
+            get { return cutoffFrequency; }
+            set
+            {
+                // set cutoffFrequency and enforce the specified range
+                if (value < 10.0f)
```

Comment:
I think we can write this a bit cleaner:
`cutoffFrequency = Mathf.Clamp(value, 10.0f, 22000.0f);`